client/components/data/query-p2-connections/index.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+/**
+ * Internal dependencies
+ */
+import { requestP2KeyringConnections } from 'calypso/state/sharing/keyring/actions';
+import { isKeyringConnectionsFetching } from 'calypso/state/sharing/keyring/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+class QueryP2Connections extends Component {
```

Comment:
Since this renders nothing – does this file need to be a `.jsx` file?  We could maybe also have this file be a functional component?

Reply:
Good point, I just copy pasted an existing query component 🙂 will update.

Reply:
Fix pushed.